server/spec/hypervisor_check_in_spec.rb

```diff
@@ -619,4 +619,122 @@ def create_virtwho_client(user)
     consumer_client = Candlepin.new(nil, nil, consumer['idCert']['cert'], consumer['idCert']['key'])
     return consumer_client
   end
+
+it 'should allow a single guest to be migrated and revoke host-limited ents' do
+    owner = create_owner random_string('test_owner1')
+    user = user_client(owner, random_string("user"))
+    virtwho = create_virtwho_client(user)
+    uuid1 = random_string('system.uuid')
+    guests = [{:guestId => uuid1}]
+
+    host_consumer = user.register(random_string('host1'), :hypervisor, nil,
```

Comment:
Any reason this consumer is a "hypervisor" while the one just below it is a "system".